infrastructure/perf/artillery.yml

```diff
@@ -0,0 +1,14 @@
+config:
+  # target: 'http://localhost:3000'
```

Comment:
This is a config file for artillery.io which is useful, though I have found https://github.com/mcollina/autocannon to be better at quickly deriving the upper limit of our request/s